types/knex.d.ts

```diff
@@ -552,7 +552,7 @@ declare namespace Knex {
     dropColumn(columnName: string): TableBuilder;
     dropColumns(...columnNames: string[]): TableBuilder;
     renameColumn(from: string, to: string): ColumnBuilder;
-    integer(columnName: string): ColumnBuilder;
+    integer(columnName: string, size:  number): ColumnBuilder;
```

Comment:
I'm curious which database you are using. Knex seems to accept this values for MSSQL and Oracle, but I can't find anywhere in their documentation that they support it. Not sure what would be the accurate name here, if there is anything in their documentation - we should use that name, but Knex itself calls it "length", not "size". In MySQL world it is "display width", but that is very MySQL-specific.

Reply:
I'm using MySQL. I will update the parameter to length.